blocks/subheadline-block/features/subheadline/default.test.jsx

```diff
@@ -19,7 +19,7 @@ describe("the subheadline feature for the default output type", () => {
 	describe("when subheadline content from globalContent is present", () => {
 		it("should dangerously set the innerHTML to the subheadline content", () => {
 			const wrapper = mount(<SubHeadline />);
-			expect(wrapper.find("h2.sub-headline").at(0).prop("dangerouslySetInnerHTML")).toStrictEqual({
+			expect(wrapper.find("h2.b-subheadline").at(0).prop("dangerouslySetInnerHTML")).toStrictEqual({
```

Comment:
I think for React Testing Library standard, it should be something like:
```suggestion
			expect(wrapper.find("SubHeadline").text()).toBe("subheadline for our story");
```
which removes the need to refactor if the class name or header hierarchy changes.  Unless what we are testing IS the hierarchy which is a concern for the HeadingSection

Reply:
Then this could be a `shallow` render if I'm not mistaken.

Reply:
I don't think the h hierarchy is intended to be tested. I updated the classname based on using the architecture

Reply:
What was the reason for resolving this comment? The tests would be less brittle by changing to the way @nschubach suggested and testing the text value of the component

Reply:
Not sure I follow. I wanted to have previous testing parity by updating the classname based on the albeit brittle structure of the previous tests. I've updated to test the entire container if that's the suggestion as I understand it?

Reply:
updated @matthewroach @nschubach based on using `text()` on the whole container.